join_array.rb

```diff
@@ -0,0 +1,6 @@
+a = Array[1,2,3,4]
+b = Array[5,6,7,8]
```

Comment:
Use spaces after `,`

Reply:
You can eliminate `Array` class here
```suggestion
a = [1, 2, 3, 4]
b = [5, 6, 7, 8]
```

Reply:
And use proper variable names.